2022Swerve/src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -99,19 +96,14 @@ public void toggleOrientation(){
     public SwerveModuleState[] convertToModuleStates(double xTranslation, double yTranslation, double rotation) {
         //Takes axis input from joysticks and returns an array of swerve module states
 
-        double x = xTranslation;
-        double y = yTranslation;
+        double x = yTranslation; //Intentional, x in swerve kinematics is y on the joystick
```

Comment:
We need to talk about this.  Our coordinate system shouldn't be based on simulation.  Unless you want to make a conditional statement here, and only do this switcharoo in simulation.

Reply:
Control x/y != Kinematic(Real World) x/y
Not a simulation concerned change

This was already approved with OI (This PR is the result of a bypass, so it includes OI commits)